typedb-src/modules/ROOT/pages/admin/update.adoc

```diff
@@ -40,13 +40,13 @@ information on TypeDB server database compatibility see the table below.
 | Database encoding Version | Compatible TypeDB Versions
```

Comment:
Why isn't "encoding" capitalised?

Reply:
I think Version(s) should be de-capitalized because we are using sentence case, usually.
Any other reasons for `encoding` to be capitalized?
```suggestion
| Database encoding version | Compatible TypeDB versions
```

Reply:
👍🏼